src/rate/demos/zhCN/index.demo-entry.md

```diff
@@ -6,13 +6,19 @@
 
 ```demo
 basic
+size
+color
+events
+icon
 ```
 
+
 ## Props
 
 | 名称            | 类型                      | 默认值      | 说明 |
 | --------------- | ------------------------- | ----------- | ---- |
 | count           | `number`                  | `5`         |      |
 | value           | `number`                  | `undefined` |      |
 | default-value   | `number`                  | `0`         |      |
+| size | `'small' \| 'medium' \| 'large'` | `'medium'` |  |
```

Comment:
Please also add a slot table. You can see doc of any component with slot as reference.